int8 obst_left
int8 obst_right
int8 obst_front
time stamp